src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -18,36 +19,41 @@
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the applicants identified by the index numbers used in the displayed person list.\n"
+            + "Parameters: INDEX... (must be positive integers)\n"
```

Comment:
Would this be better if this was written as non-negative integer, since you can input 0 as an index?

Reply:
Yes you are right! 😨

Reply:
Oh wait actually, I just realized you are not supposed to be able to input 0 since the user will be using 1-based indexing! 
😅